api-schema/xdb.yaml

```diff
@@ -328,6 +324,26 @@ components:
         - SCHEDULED_TIMER
         - FIRED_TIMER
         - SKIPPED_TIMER
+    LocalQueueResult:
+      type: object
+      required:
+        - status
+        - queueName
+      properties:
+        status:
+          $ref: '#/components/schemas/LocalQueueStatus'
+        queueName:
+          type: string
+        messages:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocalQueueMessageResult'
+    LocalQueueStatus:
+      type: string
+      enum:
+        - WAITING_LOCAL_QUEUE
+        - COMPLETED_LOCAL_QUEUE
+        - SKIPPED_LOCAL_QUEUE   # in anyOfCompletion, set this status for other waiting local queues
```

Comment:
I am not sure it's really a good idea to set it `SKIPPED` for that case. For timer status, `SKIPPED` is specific for the case where a timer is skipped intentionally by an API request : https://github.com/indeedeng/iwf-idl/blob/main/iwf.yaml#L236 

When a timer is skipped, it is equivalent as "fired" for the waitUntil. This is for mitigate some stuck timers or fire a timer earlier for testing/operation purpose. 

So when a queue command is not completed, may be just WAITING is fine to be consistent with the semantics.